ingress-inquisition/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 	fmt.Println("\n ")
 
 	var failures int = 0
-	if args.Execute {
+	if args.Execute && len(securityGroupRuleDetails) > 0 {
```

Comment:
At first I was going to comment that I don't see why this would print "No unused security groups", but then I found the line [elsewhere](https://github.com/guardian/fsbp-tools/blob/main/ingress-inquisition/utils/awsutils.go#L183)!

Would it be nice to separate printing information about the program run (which would nicely fit in our main function) from the code to lookup the data (in the awsutils module)?

I'm not sure, no dramas!